utils/check-links.js

```
@@ -0,0 +1,73 @@
+// Script to check all markdown files for broken links in non-interactive mode
+const { execSync } = require('child_process');
+const { readdirSync, statSync, writeFileSync } = require('fs');
+const { join } = require('path');
+
+// Recursively get all .md files in a directory
+const getAllMdFiles = dir => 
+  readdirSync(dir).reduce((files, file) => {
+    const name = join(dir, file);
+    if (statSync(name).isDirectory()) {
+      return [...files, ...getAllMdFiles(name)];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)